tools/halide_image.h

```diff
@@ -3,21 +3,24 @@
 // Func::compile_to_file, when you do not want to link your processing program
 // against Halide.h/libHalide.a.
 
-#ifndef _STATIC_IMAGE_H
-#define _STATIC_IMAGE_H
+#ifndef HALIDE_IMAGE_H
```

Comment:
This is the same header guard as src/Image.h. Is this intentional to prevent including both? If not you might want to make it HALIDE_TOOLS_IMAGE_H